pkg/cluster/ready.go

```diff
@@ -11,19 +11,41 @@ func (u *simpleUpgrader) WaitForInfraServices(ctx context.Context, cs *api.OpenS
 	return u.kubeclient.WaitForInfraServices(ctx)
 }
 
+func (u *simpleUpgrader) waitForNode(ctx context.Context, cs *api.OpenShiftManagedCluster, role api.AgentPoolProfileRole) error {
```

Comment:
the function name seems odd, we are waiting for all the vms in a pool
waitForPool() ?

Reply:
agreed, thanks, will fix in follow-up